lacework/resource_lacework_vulnerability_exception_container.go

```diff
@@ -402,7 +427,10 @@ func vulnerabilityExceptionReasonsSlice() []string {
 }
 
 func vulnerabilityExceptionFixableEnabled(fixable []int) bool {
```

Comment:
This code is in both, the go-sdk and the provider, we should probably move it to the go-sdk and consume it from there. https://lacework.atlassian.net/browse/ALLY-1033